<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SerialBox5_fix_baudrate" name="serial_widget" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Answers</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QRadioButton" name="rb_4">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="le_1"/>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="rb_3">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rb_2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="rb_5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="le_2"/>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="le_3"/>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="rb_1">
          <property name="text">
           <string>1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="le_4"/>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="le_5"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="le_name_1"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="le_name_2"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="le_name_3"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="le_name_4"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="le_name_5"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SerialBox5_fix_baudrate</class>
   <extends>QWidget</extends>
   <header>serialbox5_fix_baudrate.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rb_1</tabstop>
  <tabstop>le_name_1</tabstop>
  <tabstop>le_1</tabstop>
  <tabstop>rb_2</tabstop>
  <tabstop>le_name_2</tabstop>
  <tabstop>le_2</tabstop>
  <tabstop>rb_3</tabstop>
  <tabstop>le_name_3</tabstop>
  <tabstop>le_3</tabstop>
  <tabstop>rb_4</tabstop>
  <tabstop>le_name_4</tabstop>
  <tabstop>le_4</tabstop>
  <tabstop>rb_5</tabstop>
  <tabstop>le_name_5</tabstop>
  <tabstop>le_5</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
